### PR TITLE
Initial setup of mainnet DAO structure

### PR DESCRIPTION
**Pull request by TrevorJTClarke**
Dear sirs,
want eyes on DAO policy. 
Wen? ASAP
to the moon and DAO be win!

### CODE REVIEW

File: dao/dao_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
+MASTER_ACC=cron.near
+DAO_ROOT_ACC=sputnik-dao.near
+DAO_NAME=croncat
+DAO_ACCOUNT=$DAO_NAME.$DAO_ROOT_ACC
+
+##Change NODE_ENV between mainnet, testnet and betanet
+# export NODE_ENV=testnet
+export NODE_ENV=mainnet
```
Comment (mikedotexe) [inline dao/dao_deploy.sh:8]:
```suggestion
export NEAR_ENV=mainnet
```

File: dao/dao_proposals.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+set -e
+MASTER_ACC=cron.near
+DAO_ROOT_ACC=sputnik-dao.near
+DAO_NAME=croncat
+DAO_ACCOUNT=$DAO_NAME.$DAO_ROOT_ACC
+
+##Change NODE_ENV between mainnet, testnet and betanet
+# export NODE_ENV=testnet
+export NODE_ENV=mainnet
```
Comment (mikedotexe) [inline dao/dao_proposals.sh:9]:
```suggestion
export NEAR_ENV=mainnet
```

File: dao/dao_proposals.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+set -e
+MASTER_ACC=cron.near
+DAO_ROOT_ACC=sputnik-dao.near
+DAO_NAME=croncat
+DAO_ACCOUNT=$DAO_NAME.$DAO_ROOT_ACC
+
+##Change NODE_ENV between mainnet, testnet and betanet
+# export NODE_ENV=testnet
```
Comment (mikedotexe) [inline dao/dao_proposals.sh:8]:
```suggestion
# export NEAR_ENV=testnet
```

File: dao/dao_deploy.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+}'
+
+#Args for creating DAO in sputnik-factory2
+ARGS=`echo "{\"config\":  {\"name\": \"$DAO_NAME\", \"purpose\": \"Enabling a community to own grow and maintain the blockchain scheduling utility\", \"metadata\":\"\"}, \"policy\": $POLICY}" | base64`
+FIXED_ARGS=`echo $ARGS | tr -d '\r' | tr -d ' '`
+
+# Call sputnik factory for deploying new dao with custom policy
+near call $DAO_ROOT_ACC create "{\"name\": \"$DAO_NAME\", \"args\": \"$FIXED_ARGS\"}" --accountId $MASTER_ACC --amount 5 --gas 150000000000000
+near view $DAO_ACCOUNT get_policy
+echo "'$NODE_ENV' Deploy Complete!"
```
Comment (mikedotexe) [inline dao/dao_deploy.sh:83]:
```suggestion
echo "'$NEAR_ENV' Deploy Complete!"
```

File: dao/dao_deploy.sh [hunk 5]
```diff
@@ -0,0 +1,83 @@
+MASTER_ACC=cron.near
+DAO_ROOT_ACC=sputnik-dao.near
+DAO_NAME=croncat
+DAO_ACCOUNT=$DAO_NAME.$DAO_ROOT_ACC
+
+##Change NODE_ENV between mainnet, testnet and betanet
+# export NODE_ENV=testnet
```
Comment (mikedotexe) [inline dao/dao_deploy.sh:7]:
```suggestion
# export NEAR_ENV=testnet
```

File: dao/dao_deploy.sh [hunk 6]
```diff
@@ -0,0 +1,83 @@
+MASTER_ACC=cron.near
+DAO_ROOT_ACC=sputnik-dao.near
+DAO_NAME=croncat
+DAO_ACCOUNT=$DAO_NAME.$DAO_ROOT_ACC
+
+##Change NODE_ENV between mainnet, testnet and betanet
```
Comment (mikedotexe) [inline dao/dao_deploy.sh:6]:
```suggestion
##Change NEAR_ENV between mainnet, testnet and betanet
```

File: dao/dao_proposals.sh [hunk 7]
```diff
@@ -0,0 +1,18 @@
+set -e
+MASTER_ACC=cron.near
+DAO_ROOT_ACC=sputnik-dao.near
+DAO_NAME=croncat
+DAO_ACCOUNT=$DAO_NAME.$DAO_ROOT_ACC
+
+##Change NODE_ENV between mainnet, testnet and betanet
```
Comment (mikedotexe) [inline dao/dao_proposals.sh:7]:
```suggestion
##Change NEAR_ENV between mainnet, testnet and betanet
```

File: dao/README.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+
+# Are you a Croncat?
+Croncat is an organism owned & maintained by dedicated people coming from a diverse set of perspectives. 
+Here are the classifications of what makes up the types of people in Croncat DAO:
+* **Founder** - Core contributors that provide vision, implementation & leadership
+* **Agent** - The operator executing the tasks & runtime dictated by the DAO
+* **Application** - Entities that need scheduled transactions
+* **Commander** - Individuals that contribute to initiatives defined by the DAO, receive retainer stipend based on performance (Example: Growth Hacker)
+
+Becoming part of the Croncat DAO is much different than token based DAOs. It requires real interaction and participation in community development, governance, research & network growth. To maintain a seat on Croncat DAO, you believe in the vision, aspire to further the mission statement, and generally provide your personal perspective to create positive outcomes for the Croncat community as a whole. Maintaining a seat on the DAO requires interactions like voting at minimum 5 times per year.
```
Comment (mikedotexe) [inline dao/README.md:20]: beautiful 👌🏼

File: dao/README.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+# Governance & Operations
+### Council Responsibilities
+DAO council will be the sole responsible entity for the development and promotion of croncat. The council will be made up of different types of persons, each bringing a unique viewpoint to the governance process to align and balance the DAO. You can think of the council representing 3 core entities: Founders, Producers, Consumers. The council will have the power to enact proposals, fund development & marketing, provide guidance on integrations and onboard further community members.
+
+Council members are responsible to the DAO, and have a requirement to on-chain activity periodically to maintain their council seat. By staying active on the chain activities, members not only keep the community inline with their perspective, but also help create a wider base for decisions. Council members are also responsible for doing the research and diligence necessary to make sound decisions for fund allocations.
```
Comment (mikedotexe) [inline dao/README.md:26]: might expand on "on-chain activity" here in the first sentence. Doing agent stuff? Voting?

File: dao/README.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+|Application|Proposal, Voting|Proposes needs for application integrations, fees or similar. Can vote on operation & cost proposals. Early integration partners receive special swag & NFT.|
+|Agent|Proposal, Voting|Proposes needs for agent runtime, reward amount or similar. Can vote on operation & cost proposals. Early adopters receive special swag & rare NFT.|
+|Commander|Proposal|Proposes reimbursements, work initiatives, development bounties, marketing initiatives & other types of works created for furthering the growth of cron. Commanders receive differing levels of access to things like social, discussion, development session & more based on longevity and work completed.|
+
+### Proposal Types
+* **Treasury Proposal**: Items relating to or including fund or token allocation & staking accounting. 
+* **Operation Cost Proposal**: Items that pay individuals for a certain finalized development or marketing initiative.
+* **Custom Operation**: Special contract function calls that can include core runtime settings, interacting with other dApps or upgrading core contracts.
```
Comment (mikedotexe) [inline dao/README.md:40]: This is great. Another proposal type I could see:
**Enhancing incentive economics**: Proposals that identify and provide clear suggestions on how economics can be improved for the Croncat community.

File: dao/README.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+### Core Deployment
+Croncat is a living creature, developed by people and autonomously operating on the blockchain. Development will continue to be fluid, where features will be added from time to time. When a new feature is ready to be deployed, the compiled contract code will be staged on-chain, and submitted as an upgrade proposal. Core DAO members will be responsible for testing & ensuring the upgrade will not be malicious, align with all representative parties of cron DAO and meet all coding standards for production contracts. Upon successful approval of upgrade, the croncat contract will utilize a migration function to handle any/all state changes needed. In the event that there are backward incompatibilities, the DAO can decide to launch an entirely new deployed contract. This type of change will need to be communicated among all integration partnerships, publicly disclosed on social and website and maintain the legacy contract until all tasks have been completed.
```
Comment (mikedotexe) [inline dao/README.md:55]: very clear, very well written 👌🏼

**Review by mikedotexe — APPROVED** [review]
v good. v v good

File: dao/README.md (hunk 9 above)
Comment (TrevorJTClarke) [reply dao/README.md:26]: I will handle that in the docs side, this was just for reference of why the policy was set this way to start.

File: dao/README.md (hunk 10 above)
Comment (TrevorJTClarke) [reply dao/README.md:40]: Luvvit!

**Pull request merged**